oss/Dockerfile

```diff
@@ -12,40 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM       centos:centos7
-MAINTAINER Sonatype <cloud-ops@sonatype.com>
+FROM registry.access.redhat.com/ubi8/ubi
+
 LABEL vendor=Sonatype \
+  maintainer="Sonatype <cloud-ops@sonatype.com>" \
   com.sonatype.license="Apache License, Version 2.0" \
   com.sonatype.name="Nexus Repository Manager OSS base image"
 
 ARG NEXUS_VERSION=2.14.13-01
 ARG NEXUS_DOWNLOAD_URL=https://download.sonatype.com/nexus/oss/nexus-${NEXUS_VERSION}-bundle.tar.gz
 
 ENV SONATYPE_WORK=/sonatype-work
-RUN yum install -y \
-  curl tar createrepo java-1.8.0-openjdk \
```

Comment:
QUESTION: since this is Nexus Repo 2, does `createrepo` need to be there in order to run yum support? Not sure if it's installed by default with that RedHat image so asking.

Reply:
I added a comment to test this - it's possible that the package name changed to createrepo_c (according to this - https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/considerations_in_adopting_rhel_8/index there is a new package in RHEL8 called createrepo_c).

Reply:
whether createrepo is available may also be buried behind a paywall: https://access.redhat.com/solutions/3755901 :(

I suspect that the support for running a yum repo has changed in rhel8 based on the instructions being different here: https://access.redhat.com/solutions/23016